core/src/main/java/hudson/AboutJenkins.java

```diff
@@ -10,7 +12,7 @@
  * 
  * @author Kohsuke Kawaguchi
  */
-@Extension
+@Extension @Symbol("about")
```

Comment:
It is pointless to be applying symbols to random extensions like this. We only care about `Descriptor`s. So a :bug: for polluting the codebase with symbols with no known use.